app/models/effective/datatables/family_data_table.rb

```diff
@@ -57,17 +57,19 @@ class FamilyDataTable < Effective::MongoidDatatable
                                 secure_message_link_type(row, current_user)])
           end
 
-          if individual_market_is_enabled?
-            dropdown += [
-                ['Edit APTC / CSR', edit_aptc_csr_path(family_id: row.id, person_id: row.primary_applicant.person.id),
-                 aptc_csr_link_type(row, pundit_allow(Family, :can_update_ssn?))],
-                ['Paper', resume_enrollment_exchanges_agents_path(person_id: row.primary_applicant.person.id, original_application_type: 'paper'), 'static'],
-                ['Phone', resume_enrollment_exchanges_agents_path(person_id: row.primary_applicant.person.id, original_application_type: 'phone'), 'static'],
-                ['Transition Family Members', transition_family_members_insured_families_path(family: row.id, family_actions_id: "family_actions_#{row.id}"),
-                 transition_family_members_link_type(row, pundit_allow(Family, :can_transition_family_members?)) ? 'ajax' : 'disabled']
-
-            ]
-          end
+          dropdown += if individual_market_is_enabled?
```

Comment:
I'm a little rusty but what exactly are you trying to do here, remove the `transition_family_members_link_type` unless aca individual market is enabled?

It would be better to do something like

```
dropdown << ['Transition Family Members', transition_family_members_insured_families_path(family: row.id, family_actions_id: "family_actions_#{row.id}"),
                           transition_family_members_link_type(row, pundit_allow(Family, :can_transition_family_members?)) ? 'ajax' : 'disabled']
                        ] if aca_individual_market_feature_is_enabled?
 ```
Just in case more options are added